ios/AirMaps/AIRMapMarkerManager.m

```diff
@@ -33,7 +33,7 @@ - (UIView *)view
     return marker;
 }
 
-//RCT_EXPORT_VIEW_PROPERTY(identifier, NSString)
+RCT_EXPORT_VIEW_PROPERTY(identifier, NSString)
```

Comment:
@lelandrichardson is there a reason why this was commented out? I saw this https://github.com/lelandrichardson/react-native-maps/issues/37 would this resolve the issue?